### PR TITLE
Contact Form: Add New Years Day closure messages

### PR DESCRIPTION
**Pull request by mattwondra**
Adds a banner to the top of the Contact Us form if you're eligible for live chat, noting that live chat will be closed on January 1, 2018.

Ref 515-gh-hg

### To test

Fake the current date by modifying **both lines** that say `const currentDate = i18n.moment();`. Set to the following and you should get the described result:

- `const currentDate = i18n.moment();`  
  You should see the current pre-Christmas notice
- `const currentDate = i18n.moment( '2017-12-25 12:00:00' );`  
  You should see the Christmas Day notice
- `const currentDate = i18n.moment( '2017-12-26 12:00:00' );`  
  The notice should be gone
- `const currentDate = i18n.moment( '2017-12-30 12:00:00' );`  
  You should see the pre-New Year's notice
- `const currentDate = i18n.moment( '2018-01-01 12:00:00' );`  
  You should see the New Year's Day notice
- `const currentDate = i18n.moment( '2018-01-02 12:00:00' );`  
  The notice should be gone

### CODE REVIEW

File: client/me/help/help-contact/index.jsx [hunk 1]
```diff
@@ -681,16 +683,24 @@ class HelpContact extends React.Component {
 			this.getContactFormPropsVariation( supportVariation )
 		);
 
-		const currentDate = Date.now();
+		const currentDate = i18n.moment();
```
Comment (omarjackman) [inline client/me/help/help-contact/index.jsx]: Looks like you fixed a timezone bug here 👍

File: client/me/help/help-contact/index.jsx [hunk 2]
```diff
@@ -681,16 +683,24 @@ class HelpContact extends React.Component {
...
 
 		// Customers sent to Directly and Forum are not affected by the Christmas closures
 		const isUserAffectedByChristmas2017Closure =
 			supportVariation !== SUPPORT_DIRECTLY && supportVariation !== SUPPORT_FORUM;
 
+		const isClosureNoticeAvailable =
```
Comment (omarjackman) [inline client/me/help/help-contact/index.jsx]: Maybe `isClosureNoticeInEffect`?

**Comment by michelleweber** [comment]
I'm happy to review the text, but can you give me a little guidance? I'm not sure where/how to modify that code to see it. (You can also just paste it here rather than walking me through this; I don't need to see it in place.) Thanks!

**Comment by omarjackman** [comment]
@michelleweber I tagged it with needs copy review because of all the text that appears [here](https://github.com/Automattic/wp-calypso/pull/21057/files#diff-ea632e3aa8f084af5d3709ae52253ae1R26). Its totally possible that it was reviewed already but I tagged it just in case

**Comment by michelleweber** [comment]
Thanks, @omarjackman! I'd not seen this before. My feedback:
- Delete "you can" wherever it appears -- just say "If you need to get in touch with us, submit a support request..."
- Rather than "we will get to it as fast as we can" I'd say "we'll respond by email." Let them know what to expect, and don't create the expectation that it will necessarily be fast.

File: client/me/help/help-contact/index.jsx (hunk 2 above)
Comment (mattwondra) [reply client/me/help/help-contact/index.jsx]: Great suggestion, changing.

**Comment by mattwondra** [comment]
@michelleweber Thanks for the copy tweaks! The new copy for both the Christmas and New Year's notices read as:

> Live chat will be closed on Sunday, December 24th and Monday, December 25th for the Christmas holiday. If you need to get in touch with us, you’ll be able to submit a support request from this page and we'll respond by email. Live chat will reopen on December 26th. Thank you!

> Live chat is closed today for the Christmas holiday. If you need to get in touch with us, submit a support request below and we'll respond by email. Live chat will reopen on December 26. Thank you!

> Live chat will be closed on Monday, January 1, 2018 for the New Year's Day holiday. If you need to get in touch with us, you’ll be able to submit a support request from this page and we'll respond by email. Live chat will reopen on January 2nd. Thank you!

> Live chat is closed today for the New Year's Day holiday. If you need to get in touch with us, submit a support request below and we'll respond by email. Live chat will reopen on January 2nd. Thank you!


/cc @kevmarsden

File: client/me/help/help-contact-closed/index.jsx [hunk 3]
```diff
@@ -8,30 +8,52 @@ import React from 'react';
...
+	} else if ( currentDate >= christmas2017ClosureStartsAt ) {
+		closureHeading = translate( 'Limited Support over Christmas' );
+		closureMessage = translate(
+			'Live chat is closed today for the Christmas holiday. If you need to get in touch with us, submit ' +
+				"a support request below and we'll respond by email. Live chat will reopen on December 26. Thank you!"
+		);
+	} else {
+		closureHeading = translate( 'Limited Support over Christmas' );
+		closureMessage = translate(
+			'Live chat will be closed on Sunday, December 24th and Monday, December 25th for the Christmas holiday. ' +
```
Comment (a8ci18n) [inline client/me/help/help-contact-closed/index.jsx]: Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'Live chat will be closed on Sunday, December 24th and Monday, December 25th for the Christmas holiday. If you need to get in touch with us, you’ll be able to submit a support request from this page and we will get to it as fast as we can. Live chat will reopen on December 26th. Thank you!' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Pull request merged**